crates/bevy_asset/src/loader_builders.rs

```diff
@@ -0,0 +1,302 @@
+use crate::{
```

Comment:
A line of module docs would be good here.

Reply:
Oops, meant to do that and forgot, thanks!